src/lib/network/network_autoyast.rb

```diff
@@ -227,6 +227,27 @@ def valid_rename_udev_rule?(rule)
       true
     end
 
+    def rename_lan_item(item, name_to, attr = nil, key = nil)
+      return if item.nil? || item.empty? || item < 0 || item >= LanItems.Items.size
+      return if name_to.nil? || name_to.empty?
+
+      # selecting according device name is unreliable (selects only in between configured devices)
+      LanItems.current = item
+
+      if !attr.nil? && !key.nil?
```

Comment:
if only one of this argument is set, I expect argument error here, as it is obviously wrong usage.